apps/confidential/confidential_test.go

```diff
@@ -744,6 +748,9 @@ func TestNewCredFromTokenProvider(t *testing.T) {
 	if v := int(time.Until(ar.ExpiresOn).Seconds()); v < expiresIn-2 || v > expiresIn {
 		t.Fatalf("expected ExpiresOn ~= %d seconds, got %d", expiresIn, v)
 	}
+	if v := int(time.Until(ar.Metadata.RefreshOn).Seconds()); v < refreshOn-2 || v > refreshOn {
```

Comment:
Why do you need this tolerance here? And if one is needed, should it live in some helper to benefit all tests and to keep DRY?

Reply:
Updated